backend/module/eCampCore/src/Entity/ContentTypeConfig.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace eCamp\Core\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use eCamp\Lib\Entity\BaseEntity;
+
+/**
+ * @ORM\Entity
+ */
+class ContentTypeConfig extends BaseEntity implements BelongsToCampInterface {
+    /**
+     * @var ActivityCategory
+     * @ORM\ManyToOne(targetEntity="ActivityCategory")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $activityCategory;
+
+    /**
+     * @var ContentType
+     * @ORM\ManyToOne(targetEntity="ContentType")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $contentType;
+
+    /**
+     * @var string
+     */
+    private $contentTypeConfigTemplateId;
+
+    /**
+     * ContentType should be present on each activitiy.
+     *
+     * @var bool
+     * @ORM\Column(type="boolean", nullable=false)
+     */
+    private $required = false;
+
+    /**
+     * Multiple instances on a single activitiy are resonable.
+     *
+     * @var bool
+     * @ORM\Column(type="boolean", nullable=false)
+     */
+    private $multiple = true;
+
+    /**
+     * @return Camp
+     */
+    public function getCamp() {
+        return $this->activityCategory->getCamp();
+    }
+
+    /**
+     * @return ActivityCategory
+     */
+    public function getActivityCategory() {
```

Comment:
Wieso verwenden wir nicht einen Return Typ?
```suggestion
    public function getActivityCategory(): ActivityCategory {
```

Reply:
Finde den Input gut.
Es gibt aber noch weitere Stellen, welche umgebaut werden sollten.
Habe ein Issue erfasst: #838